.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "rum-events-format"]
+	path = rum-events-format
+	url = https://github.com/DataDog/rum-events-format
```

Comment:
Isn't it possible to add `rum-event-format` as a dev dependency in the package.json by using:
```
"rum-event-format": "DataDog/rum-event-format"
```
It would remove the need to run the `git submodule` command before every test.

Reply:
yes, it is. However, the behavior are not the same:

as a *node dependency*:

- a rum-events-format specific commit is linked in the yarn.lock, so we need to manually update it at each rum-events-format modification
- repo will be downloaded for each ci job

as a *git submodule*:

- we can always test to the HEAD of the repo but also be tied to a specific commit if we want to
- we can pull repo only when we need to (before the validation test)

Reply:
👌